### PR TITLE
feat(broadcast-event): add request body example

### PR DESCRIPTION
**Pull request by FlowFlorent**
Screen Shot 2023-03-15 at 1 33 33 PM

### CODE REVIEW

File: docs/references/v2.openapi.json [hunk 1]
```diff
@@ -717,10 +717,19 @@
...
+        "description": "This endpoint enables the broadcast of an event to a room without having to connect to it via the `client` from `@liveblocks/client`. It takes any valid JSON as a request body.",
         "requestBody": {
           "content": {
-            "application/json": {},
+            "application/json": {
+              "examples": {
+                "example-1": {
+                  "value": {
+                    "jobId": "process-image",
+                    "status": "completed"
```
Comment (CTNicholas) [inline docs/references/v2.openapi.json:728]:
```suggestion
                    type: "EMOJI", emoji: "🔥"
```

Comment (CTNicholas) [reply docs/references/v2.openapi.json:728]: Could we use this emoji example from the rest of the docs (e.g. [here](https://liveblocks.io/docs/api-reference/liveblocks-react#useBroadcastEvent)). Consistency is good for our users.

**Review by CTNicholas — CHANGES_REQUESTED** [review]
Ahh, thanks for starting this! I thought the API page was automatically generated from somewhere else, which is why I didn't make the change myself 👍

File: docs/references/v2.openapi.json (hunk 1 above)
Comment (FlowFlorent) [reply docs/references/v2.openapi.json:728]: Hah! That's what I went for originally, of course!

File: docs/references/v2.openapi.json [hunk 2]
```diff
@@ -717,10 +717,18 @@
...
-        "description": "This endpoint enables the broadcast of an event to a room without having to connect to it via the `client` from `@liveblocks/client`.",
+        "description": "This endpoint enables the broadcast of an event to a room without having to connect to it via the `client` from `@liveblocks/client`. It takes any valid JSON as a request body.",
         "requestBody": {
           "content": {
-            "application/json": {},
+            "application/json": {
+              "examples": {
+                "example-1": {
+                  "value": {
+                    type: "EMOJI", emoji: "🔥"
```
Comment (FlowFlorent) [inline docs/references/v2.openapi.json:727]:
```suggestion
                    "type": "EMOJI", "emoji": "🔥"
```

**Pull request merged**